example/docs/components/TransformableImage.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: CropperBackgroundImage
+title: TransformableImage
```

Comment:
There were previously three CropperBackgroundImage components in the tree. These title changes correct the confusing duplication